### PR TITLE
Check prefix length while configuring route

### PR DESCRIPTION
**Pull request by sumukhatv**
Allow only prefixes with length greater than /18 for v4 and equal to /64 for v6

### CODE REVIEW

File: go-server-server/go/default.go [hunk 1]
```diff
@@ -1229,7 +1230,29 @@ func ConfigVrouterVrfIdRoutesPatch(w http.ResponseWriter, r *http.Request) {
...
-
+        kv, err := GetKVs(confdb.db_num, generateDBTableKey(confdb.separator, VNET_TB, vnet_id_str))
+        if err != nil {
+            WriteRequestError(w, http.StatusInternalServerError, "Internal service error", []string{}, "")
+            return
+        }
+        if adv_prefix, ok := kv["advertise_prefix"]; ok {
+            if adv_prefix == "true" {
+                prefix_len, _ := network.Mask.Size()
+                if isV4orV6(ip.String()) == 4 {
```
Comment (prsunny) [inline go-server-server/go/default.go:1241]: Can we use this - `ip.To4()`

File: go-server-server/go/default.go [hunk 2]
```diff
@@ -1229,7 +1230,29 @@ func ConfigVrouterVrfIdRoutesPatch(w http.ResponseWriter, r *http.Request) {
             failed = append(failed, r)
             continue
         }
-
+        kv, err := GetKVs(confdb.db_num, generateDBTableKey(confdb.separator, VNET_TB, vnet_id_str))
```
Comment (prsunny) [inline go-server-server/go/default.go:1233]: Can we have a logic to cache this after getting it first time? We can have significantly high number of routes and for each time, checking the DB can be expensive. So suggest a simple logic to set advertisement field for a vnet.

File: go-server-server/go/default.go (hunk 1 above)
Comment (sumukhatv) [reply go-server-server/go/default.go:1236]: The `isV4orV6` method is calling `ip.To4()`

**Review by prsunny — APPROVED** [review]

**Comment by prsunny** [comment]
please update submodule for master/202012

**Pull request merged**